src/bindings/c/ov/src/ov_c_api.cpp

```diff
@@ -0,0 +1,1310 @@
+            break;
+        default:
+            break;
+        }
+        property = property->next;
+    }
+    return config;
+}
+
+ov_status_e ov_compiled_model_get_runtime_model(const ov_compiled_model_t* compiled_model,
+                                                ov_model_t **model) {
+    if (!compiled_model || !model) {
+        return ov_status_e::GENERAL_ERROR;
+    }
+
+    try {
+        *model = new ov_model_t;
+        auto runtime_model = compiled_model->object->get_runtime_model();
+        (*model)->object = std::const_pointer_cast<ov::Model>(std::move(runtime_model));
+    } CATCH_OV_EXCEPTIONS
+
+    return ov_status_e::OK;
+}
+
+ov_status_e ov_compiled_model_get_inputs(const ov_compiled_model_t* compiled_model,
+                                        ov_output_node_list_t *input_nodes) {
+    if (!compiled_model || !input_nodes) {
+        return ov_status_e::GENERAL_ERROR;
+    }
+
+    try {
+        auto inputs = compiled_model->object->inputs();
+        int num = inputs.size();
+        input_nodes->num = num;
+        input_nodes->output_nodes = new ov_output_node_t[num];
+        for (int i = 0; i < num; i++) {
+            input_nodes->output_nodes[i].object = std::make_shared<ov::Output<const ov::Node>>(std::move(inputs[i]));
+        }
+    } CATCH_OV_EXCEPTIONS
+
+    return ov_status_e::OK;
+}
+
+ov_status_e ov_compiled_model_get_outputs(const ov_compiled_model_t* compiled_model,
+                                        ov_output_node_list_t *output_nodes) {
+    if (!compiled_model || !output_nodes) {
+        return ov_status_e::GENERAL_ERROR;
+    }
+
+    try {
+        auto outputs = compiled_model->object->outputs();
+        int num = outputs.size();
+        output_nodes->num = num;
+        output_nodes->output_nodes = new ov_output_node_t[num];
+        for (int i = 0; i < num; i++) {
+            output_nodes->output_nodes[i].object = std::make_shared<ov::Output<const ov::Node>>(std::move(outputs[i]));
+        }
+    } CATCH_OV_EXCEPTIONS
+
+    return ov_status_e::OK;
+}
+
+ov_status_e ov_compiled_model_create_infer_request(const ov_compiled_model_t* compiled_model,
+                                                    ov_infer_request_t **infer_request) {
+    if (!compiled_model || !infer_request) {
+        return ov_status_e::GENERAL_ERROR;
+    }
+
+    try {
+        *infer_request = new ov_infer_request_t;
+        auto inferReq = compiled_model->object->create_infer_request();
+        (*infer_request)->object = std::make_shared<ov::InferRequest>(std::move(inferReq));
+    } CATCH_OV_EXCEPTIONS
+
+    return ov_status_e::OK;
+}
+
+ov_status_e ov_compiled_model_set_property(const ov_compiled_model_t* compiled_model,
+                                            const ov_property_t* property) {
+    if (!compiled_model || !property) {
+        return ov_status_e::GENERAL_ERROR;
+    }
+
+    try {
+        ov::AnyMap config = property2Map(property);
```

Comment:
We should use snake_case for all the code.

Reply:
sure, will do it.